modules/AlphaWalletFoundation/AlphaWalletFoundation/Transactions/Types/GetTransactionsRequest.swift

```diff
@@ -8,44 +8,12 @@
 import Foundation
 
 public struct GetTransactions: URLRequestConvertible {
-    struct NoBlockchainExplorerApi: Error {
-    }
-
-    let server: RPCServer
-    let address: AlphaWallet.Address
-    let startBlock: Int
-    let endBlock: Int
-    let sortOrder: SortOrder
-
     public enum SortOrder: String {
```

Comment:
Seems like we don't need `GetTransactions` anymore. If we want to keep the PR smaller, can just remove `asURLRequest()` and make `GetTransactions` no longer conform to `URLRequestConvertible`?

Reply:
yes, i will remove it in another pr, to avoid conflicts for now